extensions/extensions-helm/extensions-helm-spark-infrastructure/aissemble-spark-history-chart/README.md

```diff
@@ -44,36 +51,58 @@ _**NOTE:**_ *the version should match the aiSSEMBLE project version.*
+PersistentVolumes, you can set the `eventVolume.storageClass` property to the name of the StorageClass. If you're using
 Rancher Desktop and you don't want to enable cache persistence between deployments, it's possible to use Rancher's
 Local Path Provisioner to dynamically create and destroy the cache by setting `eventVolume.storageClass` to `local-path`
 and `eventVolume.storageType` to `custom`.
 
 ## Storing the Cache on the Node
+
 This option is intended for use in development environments. By setting `ivyCache.storeCacheOnNode` to true, the chart
 will create a PersistentVolume for you that maps to a local directory on the host machine of the Node. This directory
-is defined by the `eventVolume.volumePathOnNode` property.  This allows the Ivy cache to be reused between deployments.
+is defined by the `eventVolume.volumePathOnNode` property. This allows the Ivy cache to be reused between deployments.
 Because the default location is in the `/tmp` directory, the cache may still be cleared between container restarts (e.g.
 when restarting Rancher Desktop).
 
 ### Considerations
-As noted, the local storage option is intended for development environments.  This is due to the inherent security risk 
-of allowing a pod to access the host's filesystem.  In many environments, [`hostPath` volumes](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)
-are disabled by default for this reason.  Moreover, it's important to note that because hostPath and local volumes are
+
+As noted, the local storage option is intended for development environments. This is due to the inherent security risk
+of allowing a pod to access the host's filesystem. In many
+environments, [`hostPath` volumes](https://kubernetes.io/docs/concepts/storage/volumes/#hostpath)
+are disabled by default for this reason. Moreover, it's important to note that because hostPath and local volumes are
 tied directly to a single Node, this functionality may not work as expected in mult-node environments.
 
-It is possible to configure the [Pod Security admission controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/)
-to allow hostPath volumes, either across the board or for Spark History Server pods specifically.  Alternatively, a [`local`
+It is possible to configure
+the [Pod Security admission controller](https://kubernetes.io/docs/concepts/security/pod-security-admission/)
+to allow hostPath volumes, either across the board or for Spark History Server pods specifically. Alternatively,
+a [`local`
 volume](https://kubernetes.io/docs/concepts/storage/volumes/#local) can be used which alleviates most of the security
 concerns as well as the single-node limitation. However, redeployments of the Spark Operator will not reuse the same
 volume, meaning the cache will be lost between deployments.
+
+# Migration from aiSSEMBLE v1 Helm Charts
+
+If you are migrating from the v1 version of the hive-metastore-service chart, use the tables below to apply any
```

Comment:
This should refer to the v1 spark-infrastructure chart

Reply:
yes